utils/gitlog2changelog.py

```diff
@@ -76,7 +76,7 @@
             date = dateList[1]
             date = date[0 : len(date) - 1]
             dateFound = True
-        except:
+        except (IndexError, TypeError):
```

Comment:
Why TypeError?

Reply:
TypeError: This will occur if dateList is not a list or if dateList[1] is not a string. This can happen if the input to this code snippet is not as expected.

Reply:
dateList and dateList[1] are a result of the [re.split](https://docs.python.org/3/library/re.html#re.split) function call on line 74 which returns a list of strings according to the documentation.